packages/@glimmer/runtime/test/initial-render-test.ts

```diff
@@ -54,6 +54,22 @@ class AbstractRehydrationTests extends InitialRenderSuite {
 
 class Rehydration extends AbstractRehydrationTests {
 
+  @test "rehydrates into element with pre-existing content"() {
+    let rootElement = this.delegate.serverEnv.getAppendOperations().createElement('div') as HTMLDivElement;
```

Comment:
This and the line below it are changed from the original test implementation to use the `serverEnv` rather than the `clientEnv` to ensure we get the proper serialization markers.